etc/galasa-tests/com.ibm.cics.cip.cbsa.galasa.tests.manager/src/main/java/com/ibm/cics/cip/cbsa/galasa/tests/manager/internal/CicsRegionFactory.java

```diff
@@ -0,0 +1,92 @@
+package com.ibm.cics.cip.cbsa.galasa.tests.manager.internal;
+
+import javax.validation.constraints.NotNull;
+
+import dev.galasa.ProductVersion;
+import dev.galasa.cicsts.CicstsManagerException;
+import dev.galasa.cicsts.MasType;
+import dev.galasa.cicsts.spi.BaseCicsImpl;
+import dev.galasa.cicsts.spi.ICicstsManagerSpi;
+import dev.galasa.zos.IZosImage;
+import dev.galasa.zosbatch.IZosBatchJob;
+
+public class CicsRegionFactory extends BaseCicsImpl {
```

Comment:
I can't see where this CicsRegionFactory class is being used and it looks like it doesn't support any operations?